### DISCUSSION TITLE
Upgrade to 6.0, No hint path defined for [backpack]

### DISCUSSION THREAD

**Discussion by lukepower** [Laravel-Backpack/community-forum#554] [Q&A (Help)] [answered]
Hi there,

I went through the upgrade from 5.4 to 6.0 follwing the Upgrade guide.
All went flawlessy, however I am now facing an issue that while looking at error pages there is an error popping up:

`No hint path defined for [backpack] `

It seems to be related with this line in the view file:

`@extends(backpack_user() && (Str::startsWith(\Request::path(), config('backpack.base.route_prefix'))) ? 'backpack::layouts.top_left' : 'backpack::layouts.plain')`

I do wonder, is there a need to update those templates as well?

Best regards
 Luke

**Comment by pxpm** [comment, marked as answer]
Hey @lukepower 

Those error templates were removed from user land. 

We now only handle errors with theme template for http errors inside admin panel. 

All the rest is handled by the framework errors and you can style them as you see fit. 

I will make sure we mention this in the upgrade guide 👍

**Reply by lukepower** [reply]
@pxpm thanks, that solved it! :)
